web-client/switch-colors.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+[ -z "${ENV}" ] && echo "You must have ENV set in your environment" && exit 1
```

Comment:
for when this is run locally, it's helpful to short circuit if the environment isn't properly configured